eisvogel.latex

```diff
@@ -458,12 +458,22 @@ $endif$
 \PassOptionsToPackage{hyphens}{url}
 
 %
-% When using babel or polyglossia with biblatex, loading csquotes is recommended 
+% When using babel or polyglossia with biblatex, loading csquotes is recommended
 % to ensure that quoted texts are typeset according to the rules of your main language.
 %
 \usepackage{csquotes}
 
 %
+% Quotes
+%
+\usepackage{framed}
+\usepackage{xcolor}
+\let\oldquote=\quote
+\let\endoldquote=\endquote
+\colorlet{shadecolor}{blue!15}
```

Comment:
I think the blue is a bit intensive here. Could you use 

```
\colorlet{shadecolor}{quote-background}
```
and define a color `quote-background` in [line 421](https://github.com/Wandmalfarbe/pandoc-latex-template/pull/11/commits/751705d804cb083fbdc192c4598077a0d163a639#diff-0aa458022c64e642926b2ded90215a09R421) like this:

```
\colorlet{quote-background}{listing-keyword!15}
```